frontend/src/scenes/Workspace/reducers/reducer.ts

```diff
@@ -1,21 +1,26 @@
 import { Routine } from 'redux-saga-routines';
-import { addWorkspaceRoutine, selectChatRoutine } from '../routines/routines';
+import { addWorkspaceRoutine, selectChatRoutine, selectWorkspaceRoutine } from '../routines/routines';
 import { IWorkspace } from 'common/models/workspace/IWorkspace';
+import { IChat } from 'common/models/workstate/chat';
 
 export interface IWorkspaceState {
   workspace: IWorkspace;
   loading: boolean;
-  error: any;
-  selectedChat: any;
-  channels: Array<any>;
-  directMessages: Array<any>;
+  error: string;
```

Comment:
incorrect, it should be custom error model

Reply:
Done

Reply:
it's still string